include/seqan3/alphabet/structure/dot_bracket3.hpp

```diff
@@ -0,0 +1,318 @@
+//                 SeqAn - The Library for Sequence Analysis
+// ============================================================================
+//
+// Copyright (c) 2006-2017, Knut Reinert & Freie Universitaet Berlin
+// Copyright (c) 2016-2017, Knut Reinert & MPI Molekulare Genetik
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//     * Redistributions of source code must retain the above copyright
+//       notice, this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//     * Neither the name of Knut Reinert or the FU Berlin nor the names of
+//       its contributors may be used to endorse or promote products derived
+//       from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL KNUT REINERT OR THE FU BERLIN BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+// LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// ============================================================================
+
+/*!\file
+ * \ingroup structure
+ * \author Joerg Winkler <j.winkler AT fu-berlin.de>
+ * \brief Contains the dot bracket format for RNA structure.
+ */
+
+#pragma once
+
+#include <cassert>
+#include <string>
+#include <vector>
+
+#include <seqan3/alphabet/structure/concept.hpp>
+
+// ------------------------------------------------------------------
+// dot_bracket3
+// ------------------------------------------------------------------
+
+namespace seqan3
+{
+
+/*!\brief The three letter RNA structure alphabet of the characters ".()".
+ * \ingroup structure
+ *
+ * \details
+ * The brackets denote RNA base pair interactions. Every left bracket must have a corresponding right bracket.
+ * Pseudoknots cannot be expressed in this format. A dot (.) represents a character that is not paired.
+ *
+ *```console
+ *     GCGGAUUUAGCUCAGUUGGGAGAGCGCCAGACUGAAGAUUUGGAGGUCCUGUGUUCGAUCCACAGAAUUCGCA
+ *     (((((((..((((........)))).((((.........)))).....(((((.......)))))))))))).
+ *```
+ *
+ * \par Usage:
```

Comment:
No colon in the section header. Also the other documentation pages call this `\par Example`